test/remote/gateways/remote_merchant_e_solutions_test.rb

```diff
@@ -200,6 +201,12 @@ def test_successful_purchase_with_3dsecure_params
     assert_equal 'This transaction has been approved', response.message
   end
 
+  def test_successful_verify
```

Comment:
Why no tests for `store`?

Reply:
store is already being tested with complete flow from store to unstore operations here: https://github.com/activemerchant/active_merchant/pull/4507/files#diff-050b74195cc97ae48f83123bfd24c42b3f5354155574bda9841bebb728c76f8fL71

should I write any individual test for store as well?